src/java.desktop/macosx/classes/com/apple/laf/AquaFileChooserUI.java

```diff
@@ -325,6 +326,7 @@ protected void installStrings(final JFileChooser fc) {
         chooseButtonText = getString("FileChooser.chooseButtonText", "Choose");
         newFolderButtonText = getString("FileChooser.newFolderButtonText", "New");
         newFolderTitleText = getString("FileChooser.newFolderTitleText", "New Folder");
+        chooseButtonToolTipText = getString("FileChooser.chooseButtonToolTipText", "Choose");
```

Comment:
I suggest having `chooseButtonText` and `chooseButtonToolTipText` next to each other.

The tooltip should be `"Choose selected file"` to be consistent with tooltips for Open and Save buttons.

I also suggest adding `FileChooser.chooseButtonToolTipText` property into `aqua.properties` so that it can be properly localised.